src/AzureExtension/DevBox/Constants.cs

```diff
@@ -240,4 +246,29 @@ public static class DevBoxPowerStates
     /// Windows App, used for remote connections, Package Family Name
     /// </summary>
     public const string WindowsAppPackageFamilyName = "MicrosoftCorporationII.Windows365_8wekyb3d8bbwe";
+
+    public static readonly string LogFolderName = "Logs";
+
+    private static readonly Lazy<string> _logFolderRoot = new(GetLoggingPath);
+
+    public static readonly string LogFolderRoot = _logFolderRoot.Value;
+
+    public static readonly string OperationsDefaultErrorMsg = Resources.GetResource(DevBoxUnableToPerformOperationKey, LogFolderRoot);
+
+    public static readonly string StartMenuPinnedStatusErrorMsg = Resources.GetResource(DevBoxUnableToCheckStartMenuPinning, LogFolderRoot);
+
+    public static readonly string TaskbarPinnedStatusErrorMsg = Resources.GetResource(DevBoxUnableToCheckTaskbarPinning, LogFolderRoot);
+
+    private static string GetLoggingPath()
+    {
+        try
+        {
+            // This will always result in a returned value in non test scenarios. But will throw when run in a unit test as ApplicationData does not exist.

```

Comment:
nit: You could catch that exception and use Path.GetTempPath() to account for test scenarious.